### DISCUSSION TITLE
ganache for windows

### DISCUSSION THREAD

**Discussion by chainflash** [Cyfrin/foundry-full-course-cu#2735] [General]
I installed the ganache software for Windows and also connected my vscode to Ubuntu
If so, can I use ganache instead of anvil, and does anyone have experience with this?

**Comment by pollersan** [comment]
You can use Ganache but it is not recommended, it is very outdated, it has a very good UI but it is better to get used to Anvil imo.

**Comment by EngrPips** [comment]
Indeed.
